### PR TITLE
Arm64 package is available

### PR DESCRIPTION
**Pull request by bebound**
https://github.com/Azure/azure-cli/pull/24180 is merged and we'll release ARM64 packages.

### CODE REVIEW

File: docs-ref-conceptual/includes/cli-install-linux-apt.md [hunk 1]
```diff
@@ -59,7 +57,7 @@ If you prefer a step-by-step installation process, complete the following steps
     sudo chmod go+r /etc/apt/keyrings/microsoft.gpg
     ```
 
-3. Add the Azure CLI software repository:
+3. <div id="set-release"/>Add the Azure CLI software repository:
```
Comment (bebound) [inline docs-ref-conceptual/includes/cli-install-linux-apt.md:60]: Revert #3505 to get rid of `Line 97, Column 93: [Warning: bookmark-not-found] Cannot find bookmark '#set-release' in 'docs-ref-conceptual/includes/cli-install-linux-apt.md'.` warning.

File: docs-ref-conceptual/includes/cli-install-linux-apt.md [hunk 2]
```diff
@@ -21,8 +21,10 @@ The `apt` package manager contains an x86_64 package for the Azure CLI that has
 
 > [!WARNING]
 > Ubuntu 20.04 (Focal Fossa) and 20.10 (Groovy Gorilla) include an `azure-cli` package with version `2.0.81` provided by the `universe` repository. This package is outdated and not recommended. If this package is installed, remove the package before continuing by running the command `sudo apt remove azure-cli -y && sudo apt autoremove -y`.  See [Ubuntu package management](https://ubuntu.com/server/docs/package-management) or [ask ubuntu](https://askubuntu.com/search?q=apt+autoremove) for more information on `apt remove`.
+
+> [!NOTE]
 >
-> The `azure-cli` deb package does not support ARM64 architecture.
+> The `azure-cli` package supports ARM64 architecture from `2.46.0`.
```
Comment (jiasli) [inline docs-ref-conceptual/includes/cli-install-linux-apt.md:27]: Consider editing L12 to reflect this:

> The `apt` package manager contains an x86_64 package for the Azure CLI that has been tested on the following distributions.

Comment (jiasli) [reply docs-ref-conceptual/includes/cli-install-linux-apt.md:27]: Make sure to mention bionic has no ARM 64 support due to PMC limitation.

**Review by yonzhan — APPROVED** [review]

**Review by ChaseCrum — APPROVED** [review]
Looks good to me

**Comment by dbradish-microsoft** [comment]
Merging with one engineering and one doc author's approval.

**Pull request merged**